scripts/lib.star

```diff
@@ -596,34 +626,29 @@ def package_step(edition, ver_mode, variants=None, is_downstream=False, build_ta
         else:
             build_no = '$${SOURCE_BUILD_NUMBER}'
         cmds = [
-            '{}./bin/grabpl package --jobs 8 --edition {}{} '.format(test_args, edition, build_tags_str) + \
+            '{}./bin/grabpl package --jobs 8 --edition {} '.format(test_args, edition) + \
                 '--build-id {} --no-pull-enterprise{}{}'.format(build_no, variants_str, sign_args),
         ]
 
     return {
         'name': 'package' + sfx,
         'image': build_image,
         'depends_on': [
-            'build-backend' + sfx,
-            'build-frontend',
-            'build-plugins',
-            'test-backend' + sfx,
-            'test-frontend',
-            'codespell',
-            'shellcheck',
-            'check-dashboard-schemas',
+            # This step should have all the dependencies required for packaging, and should generate
+            # dist/grafana.version
+            'gen-version',
         ],
         'environment': env,
         'commands': cmds,
     }
 
-def e2e_tests_server_step(edition, build_tags=None, port=3001):
+def e2e_tests_server_step(edition, port=3001):
     sfx = ''
     package_file_pfx = ''
-    if build_tags:
-        sfx = '-' + '-'.join(build_tags)
+    if edition == 'enterprise2':
+        sfx = '-{}'.format(edition)
         package_file_pfx = 'grafana' + sfx
-    elif edition == 'enterprise':
+    elif edition in ('enterprise', 'enterprise2'):
```

Comment:
You don't need to change this condition as you're already checking for 'enterprise2' above.

Reply:
@AgnesToulet Thanks, will check.

Reply:
@AgnesToulet Confirmed and fixed.